tests/models/test_cat_model.py

```diff
@@ -297,3 +297,36 @@ async def test_find_many(
     )
 
     assert found_cat_summaries == expected_cat_summaries
+
+
+@pytest.mark.parametrize(
+    "existing_cat_documents, cat_url, expected_cat, cat_id",
+    [
+        (
+            [
+                {
+                    "_id": ObjectId("000000000000000000000101"),
+                    "name": "Sammybridge Cat",
+                    "ctime": datetime(2020, 1, 1, 0, 0, tzinfo=UTC),
+                    "mtime": datetime(2020, 1, 1, 0, 0, tzinfo=UTC),
+                },
+            ],
+            dto.PartialUpdateCat(
+                url="http://placekitten.com/200/300",
+            ),
+            dto.UpdateResult(matched_count=0, modified_count=0),
```

Comment:
If you update an existing cat, the update result should have matched- and modified count of 1, no?

Reply:
yes